src/index.html

```diff
@@ -80,15 +78,17 @@ <h1>Instant, themed Twitch chat overlays</h1>
 	const outputTemplate = document.querySelector("#output-template");
 	const preview = document.getElementById('preview');
 	const allFields = document.querySelectorAll('[name]');
-	overlayBuilderForm.addEventListener('submit', handleSubmit);
 	allFields.forEach((field) => field.addEventListener('change', updatePreview));
+	// Fill in the URL section under the form with the default URL
+	showURL();
+	// Update the URL section when a form input changes
+	overlayBuilderForm.addEventListener('input', showURL);
```

Comment:
Hm... should this event listener be consolidated with the `allFields.forEach((field) => field.addEventListener('change', updatePreview));` above?

Reply:
Done!
The change event and the input event fire at subtly different moments, that was quite confusing 😅 .
Now the last input field you typed in has to fire the blur event before the url updates the final time.

Reply:
Unresolving so we can maybe discuss the user experience. Which user experience would you expect more — URL updates as you type in the boxes, or URL updates whenever you leave each box?

Reply:
The only little hiccup I had with the change-event was when I finished filling in the form and clicked the copy button, I had to click it twice before it worked. (once to unfocus the form, once to click the button).

Reply:
Oooh, that's a good point - that seems pretty unintuitive behavior. For that alone, I think it might be worthwhile to split these back out into one `change` handler and one `input` handler.